Formula/pngquant.rb

```diff
@@ -13,14 +14,14 @@ class Pngquant < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "rust" => :build
   depends_on "libpng"
   depends_on "little-cms2"
 
   def install
-    system "cargo", "build", "--release"
-    bin.install "target/release/pngquant"
-    man1.install "pngquant.1"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+    lib.install "lib/libimagequant.a"
```

Comment:
Is this not installed by `make install`? If not, and you think it should be installed, then you'll need to open an upstream issue requesting it be installed and add a link to the issue here.

Reply:
`make install` does install the man file.

Reply:
not the static library, though?

Reply:
Typically `--enable-static` is a configure option if the static library is supposed to be installable.